packages/protocol/contracts/layer1/preconf/iface/IPreconfSlasher.sol

```diff
@@ -0,0 +1,144 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.24;
+
+import { ITaikoInbox } from "src/layer1/based/ITaikoInbox.sol";
```

Comment:
```suggestion
import "src/layer1/based/ITaikoInbox.sol";
```

Reply:
We didn't enforce this rule, so better to be consistent with the current styling.